data/5e/wizards_of_the_coast/core/players_handbook/ph_profs_weapon.lst

```diff
@@ -32,7 +32,7 @@ Morningstar			TYPE:Weapon.Martial
 Pike				TYPE:Weapon.Martial
 Rapier			TYPE:Weapon.Martial
 Scimitar			TYPE:Weapon.Martial
-Shortsword			TYPE:Weapon.Martial
+Shortsword			TYPE:Weapon.Martial.Monk
```

Comment:
Shortsword gets a double bonus in certain circumstances, this was removed to follow the fix the pcgen team implemented.